src/chrome/content/rules/WP-Engine.xml

```diff
@@ -69,7 +70,19 @@
 		<test url="http://www.wpengine.com/" />
 		<test url="http://www.wpengine.com/blog" />
 
-
+	<!-- deeper domains cause certificate mismatch error -->
+	<exclusion pattern="^http://([\w.-]+)\.([\w-]+)\.wpengine\.com"/>
+
+		<test url="http://talkdesk.staging.wpengine.com/"/>
+		<test url="http://kelvin.staging.wpengine.com/"/>
+		<test url="http://cfps.staging.wpengine.com/"/>
+		<test url="https://talkdesk.staging.wpengine.com/"/>
+		<test url="https://kelvin.staging.wpengine.com/"/>
+		<test url="https://cfps.staging.wpengine.com/"/>
+		
+	<!-- https links from other pages to these will cause MCB for important elements, hence the downgrades -->
```

Comment:
Can you confirm there are other pages that link to the HTTPS version of these domains?

I ask because we'd really like to decrease the number of downgrade rules, since they make certain optimizations harder (i.e. listening only on HTTP URLs).